test/e2e/ansible/playbooks/device_lifecycle.yml

```
@@ -0,0 +1,184 @@
+    device_name: ansible-integration-test-device
+    device_name_with_label_1: ansible-integration-test-device-label-1
+    device_name_with_label_2: ansible-integration-test-device-label-2
+    fleet_name: ansible-integration-test-fleet
+    connection_info: &connection_info
+      flightctl_token: "{{ flightctl_token | default(omit)}}"
+      flightctl_host: "{{ flightctl_host }}"
+      flightctl_validate_certs: False
+
+  tasks:
+    - block:
+      - name: Create a test fleet
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)